test/tests/amqp/test_amqp_racknode.py

```diff
@@ -0,0 +1,205 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+
+'''
+from time import sleep
+from datetime import *
+import threading
+import pika
+import logging
+import flogging
+import fit_common
+import test_api_utils
+from nose.plugins.attrib import attr
+
+amqp_message_received = False
+routing_key = ""
+amqp_body = ""
+logs = flogging.get_loggers()
+
+
+class AmqpWorker(threading.Thread):
+    '''
+    This AMQP worker Class will creat another thread when initialized and runs asynchronously.
+    The externalcallback is the callback function entrance for user.
+    The callback function will be call when AMQP message is received.
+    Each test case can define its own callback and pass the function name to the AMQP class.
+    The timeout parameter specify how long the AMQP daemon will run. self.panic is called when timeout.
+    eg:
+    def callback(self, ch, method, properties, body):
+        logs.debug(" [x] %r:%r" % (method.routing_key, body))

```

Comment:
This should be logs.data_log.debug()
Also, there appear to be duplicate lines here.